types/beacon_types.go

```diff
@@ -22,6 +22,10 @@ var (
 	DomainContributionAndProof        = [4]byte{0x09, 0x00, 0x00, 0x00}
 
 	DomainError = [4]byte{0x99, 0x99, 0x99, 0x99}
+
+	SyncCommitteeSize                    uint64 = 512
```

Comment:
Do we need those contestants here? or is it implementation specific

Reply:
removed (moved to implementation)